pkgs/tools/typesetting/asciidoc/default.nix

```diff
@@ -255,6 +255,7 @@ stdenv.mkDerivation rec {
   '';
 
   preInstall = "mkdir -p $out/etc/vim";
+  makeFlags = if stdenv.isCygwin then "DESTDIR=/." else null;
```

Comment:
It's only asciidoc. Without this "fix" it produces a "//" at the beginning of the path which on cygwin is translated to "\\" which is the beginning of network SAMBA path.